### DISCUSSION TITLE
Proposal: Rename Jujutsu to Jojo

### DISCUSSION THREAD

**Discussion by maxsu** [jj-vcs/jj#2520] [General]
## Proposal
Rename `jj`'s full name from Jujutsu to Jojo.
![image](https://user-images.githubusercontent.com/25123/164576641-3a63c0b6-a60e-4086-9e13-cfe0d4879c37.png)
Reasons Pro
```diff
+ Jojo is shorter and easier to write, say, and remember than Jujutsu
+ Jojo memes are based
```

Reasons Con
```diff
- Naming a DVCS after an anime might clash /w the author's style
```

## Summary 
To tap into the great love of Jojo and his Bizarre Adventures, and to make `jj`'s full name easier to remember, we should rename it to Jojo.

**Comment by martinvonz** [comment]
Heh, fine with me. Maybe people are less familiar with jujutsu than I thought. I actually renamed it from "jujube" (a fruit) because it seemed most people had never heard of that. We can have a vote when we get closer to 1.0, I guess :)

**Comment by langfield** [comment]
This is one user's (opinionated) vote for sticking with "jujutsu".

**Comment by emilypi** [comment]
I'm on board for spurious non-technical reasons (specifically, the equivalent of a git clone potentially being named `za warudo`).

**Comment by BatmanAoD** [comment]
I've only ever seen the martial art spelled "jiu-jitsu", so I thought the name _was_ based on an anime, Jujutsu Kaisen.

**Comment by necauqua** [comment]
I really like it being jujutsu, so if any sort of vote like that happens I'm almost certainly sticking with it :)

Also I feel like the name is starting to get relatively known, idk

**Comment by ocharles** [comment]
I would rather we keep it jujutsu, as there is plenty of third-party writing about it. It would be a shame to lose that connection while the underlying model is the same.

**Comment by martinvonz** [comment]
> I've only ever seen the martial art spelled "jiu-jitsu"

So had I until I went to check the spelling before I updated the documentation. I think I just googled it and landed on https://en.wikipedia.org/wiki/Jujutsu.

> I really like it being jujutsu, so if any sort of vote like that happens I'm almost certainly sticking with it :)

Thanks for the input. The more time that passes with the Jujutsu name, the more likely it is that we'll keep it, I'd say. I'm not sure if @maxsu was serious about renaming it to Jojo anyway. Using a name that's probably copyrighted seems risky.

**Comment by thoughtpolice** [comment]
We could always get a little cheeky and call the (inevitable) server-side part "Jojo", and the client side can be "Jujutsu", both under the `jj` umbrella name. Sort of like how Phabricator had a different ship-name for every major component. Except ours would (apparently) be anime(?)

**Comment by langfield** [comment]
I mean this in the most lighthearted sense, but let us not take naming advice from Phabricator.

**Comment by maxsu** [comment]
I've been pondering the idea of creating a Jojo porcelain!

The main reasons I have for wanting an SCM named Jojo are:

1. I could use `jojo pose <file>` 🚶‍♂️ to perform staging in a glamorous way ✨

2. I could introduce `jojo stand <stand-name>` 🦸‍♂️ to perform jaw-dropping [custom techniques](https://jojowiki.com/List_of_Stands) 🪄

3. The love of the Jojo community might fuel Jojo fans to embrace source code control 📝

I can achieve these objectives with a 'Jojo' theme, mod-pack, or porcelain. This would effectively minimize any copyright risk to the `jj` upstream project.  

Therefore, I will close this proposal. 

I hope to start on `jojo` over the holidays. 🌈🎅

**Reply by ocharles** [reply]
I think you might be able to do everything you want with just `jj` aliases and shell aliases, so this might be a nice easy project!
